REQUIREMENTS.txt.in

```diff
@@ -5,7 +5,6 @@ numpy != 1.13.0
 pandas >= 0.23.4
 pycodestyle<2.4.0
 pytest>=3.6.3
-regional
```

Comment:
Until a make target is in place to keep everything in sync, either a josh-ping on requirement changes or a modification of `environment.yml` would be appreciated.

Reply:
There is, but it likely changes the purpose of this. As it stands, the file reproduces what will be installed by using `conda install -c bioconda starfish`. If it sources requirements, it will match (far better) the current source code. I _haven't_ found a way for bioconda's meta.yaml to just re-use some existing definition. I _can_ add the generation code for bioconda to `spacetx/starfish` though. Thoughts on a `make release` target?

Reply:
What do you envision in a `make release` target?

Reply:
* run extended `check_requirements` to sync conda
 * bump version number
 * commit, tag, push tag
 * add `.dev` to version number, commit, and push to master
 * build package and push to pypi
 * build docker and push to hub

Reply:
migrated to https://github.com/spacetx/starfish/issues/761